app/ts/background/simulationModeHanders.ts

```diff
@@ -32,35 +32,32 @@ export async function getTransactionReceipt(ethereumClientService: EthereumClien
 	return { result: EthTransactionReceiptResponse.serialize(await getSimulatedTransactionReceipt(ethereumClientService, simulationState, request.params[0])) }
 }
 
-function getFromField(websiteTabConnections: WebsiteTabConnections, simulationMode: boolean, transactionFrom: bigint | undefined, getActiveAddressForDomain: (websiteAccess: WebsiteAccessArray, websiteOrigin: string, settings: Settings) => bigint | undefined, socket: WebsiteSocket, settings: Settings) {
+function getFromField(websiteTabConnections: WebsiteTabConnections, simulationMode: boolean, transactionFrom: bigint | undefined, activeAddress: bigint | undefined, socket: WebsiteSocket) {
```

Comment:
Consider re-arranging these parameters so you can have optional parameters if that is the intent.  My personal policy on parameter ordering is that "dependencies" should come first, followed by "parameters".  This is driven largely by the fact that this makes optional parameters possible.
```suggestion
function getFromField(websiteTabConnections: WebsiteTabConnections, socket: WebsiteSocket, simulationMode: boolean, transactionFrom?: bigint, activeAddress?: bigint) {
```